tests/util.go

```diff
@@ -171,9 +179,12 @@ func NewEnvFromMap(state *state.StateDB, envValues map[string]string, exeValues
 	env.gasLimit = common.Big(envValues["currentGasLimit"])
 	env.Gas = new(big.Int)
 
+	env.evm = vm.New(env, nil)
```

Comment:
I think that's acceptable considering the isolation of the jump table (I think this is where the time is now spend)